source/math/FloatSize2D.ooc

```diff
@@ -33,9 +33,9 @@ FloatSize2D: cover {
 	init: func@ ~square (length: Float) { this width = this height = length }
 	init: func@ ~default { this init(0.0f, 0.0f) }
 	pNorm: func (p: Float) -> Float {
-		p == 1 ?
-		this width abs() + this height abs() :
-		(this width abs() pow(p) + this height abs() pow(p)) pow(1 / p)
+		p == 1.0f ?
```

Comment:
Don't do equal comparisons with floats, compare the difference with absolute or relative error margin.